src/libs/brand-utils.test.ts

```diff
@@ -1,37 +1,50 @@
 import { getEnabledBrand, isBrand } from 'src/libs/brand-utils'
-import { brands, defaultBrand } from 'src/libs/brands'
+import { BrandConfiguration, brands, defaultBrand } from 'src/libs/brands'
+import { configOverridesStore } from 'src/libs/state'
 
 
 describe('brand-utils', () => {
-  let location
+  const testBrand: BrandConfiguration = {
+    name: 'Terra Test',
+    queryName: 'terra test',
+    welcomeHeader: 'Welcome to Terra Unit Tests',
+    description: 'This is only a test.',
+    hostName: 'testbrand.terra.bio',
+    docLinks: [],
+    logos: {
+      color: '',
+      white: '',
+    },
+  }
+
+  let originalLocation
 
   beforeAll(() => {
-    location = window.location
-    delete window.location
+    originalLocation = window.location
   })
 
   afterAll(() => {
-    window.location = location
+    window.location = originalLocation
   })
 
   describe('isBrand', () => {
     it('returns true if hostname matches brand', () => {
       // Arrange
-      window.location = new URL('https://testbrand.terra.bio/path/to/page')
+      window.location = new URL('https://testbrand.terra.bio/path/to/page') as unknown as Location
```

Comment:
This would probably be more easily testable if brand-utils accessed window.location through a helper that could be mocked, but I'm trying to minimize functional code changes in this PR.

Reply:
Went ahead and did this in 588f706920681a473dfe1b16873bfca93cecbec8.